pkg/skaffold/runner/debugging.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package runner
+
+import (
+	"io"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/kubectl"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/kubernetes/debugging"
+)
+
+// isDebugMode returns true if we're running for debugging.
+func (r *SkaffoldRunner) isDebugMode() bool {
+	return r.runCtx.Opts.Command == "debug"
+}
```

Comment:
I introduced this function to test whether Skaffold was launched as `skaffold debug`. This works for debug, but it doesn't seem a great pattern — for example, debug effectively invokes dev under the hood.

Reply:
nit: we already have a `runContext.DevMode` that serves a similar purpose. Can we make this similar? I don't have a preference towards either, I just don't want inconsistency.